build-a-game-playing-agent/agent_test.py

```diff
@@ -95,18 +94,16 @@ def testWrapper(self):
                     raise err[0](err[1]).with_traceback(err[2])
                 return res
             except QueueEmptyError:
-                raise TimeoutError("Test aborted due to timeout. Test was " +
-                    "expected to finish in less than {} second(s).".format(time_limit))
+                raise TimeoutError("Test aborted due to timeout. Test was "
+                                   "expected to finish in less than {} second(s).".format(time_limit))
 
         return testWrapper
 
     return wrapUnitTest
 
 
 def makeEvalTable(table):
-
```

Comment:
Seems like some of the others (e.g., 109) would be covered by PEP257: "Insert a blank line after all docstrings (one-line or multi-line) that document a _class_ -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line."